includes/node/multi_queue_node.h

```diff
@@ -3,6 +3,8 @@
 #include "cris/core/message/lock_queue.h"
 #include "cris/core/node/node.h"
 
+#include <boost/functional/hash.hpp>
```

Comment:
Don't forget to link in bazel

Reply:
Seems it is header-only. I did not see documents mentioned it so you may double-check